github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -400,6 +400,31 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: false
+    cluster: prow-s390x-workloads
+    decorate: true
+    decoration_config:
+      timeout: 3h0m0s
+    labels:
+      preset-docker-mirror-proxy: "true"
```

Comment:
Sorry - I missed this in the initial review - the s390x cluster doesn't have the image proxy so this preset will cause the job to fail to schedule. 
 
Can you remove this preset in a follow up?

Reply:
@brianmcarey 

I've created this PR 3684

Do we've to remove this label also for publish provider jobs for s390x at #3566 ?

Reply:
@brianmcarey Any doc which I can read to know more about docker mirror proxy use-case we use it for?

Reply:
This doc gives an overview of the services we use on the main clusters - https://github.com/kubevirt/project-infra/blob/main/docs/infrastructure-components.md